stock_by_warehouse_stock/models/stock_move.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2020 Vauxoo
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import json
+from odoo import api, fields, models
+
+
+class StockMove(models.Model):
+    _inherit = 'stock.move'
+
+    warehouses_stock_location = fields.Text(store=False, readonly=True)
```

Comment:
This field is already definde on `stock_by_warehouse`, isn't it?

Reply:
@CarmenMiranda good catch here from luis, it is not necessary to redefine the field that the abstract is considering, please review.

Reply:
I already talked with @luisg123v and we agreed that I'm defining them on the `stock.move` not on the `product.product` model and I need them in order to show them on the stock move widget.